alerting_alert_rule.go

```diff
@@ -0,0 +1,95 @@
+package gapi
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// AlertRule represents a Grafana Alerting Alert Rule.
```

Comment:
```suggestion
// AlertRule represents a Grafana Alert Rule.
```
Nitpick. This is inconsistent with the other comments but reads a bit better IMO.